client.js

```diff
@@ -1,31 +1,47 @@
-function u8tohex (arr) {
-  var vals = [ '0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f' ]
-  var ret = ''
-  for (var i = 0; i < arr.length; ++i) {
-    ret += vals[(arr[i] & 0xf0) / 0x10]
-    ret += vals[(arr[i] & 0x0f)]
+function MonetizerClient (opts) {
+  var domain = new URL(window.location).origin
```

Comment:
nitpick, but could do `domain = (opts && opts.url) || new URL(window.location).origin` like the cookieName option